### ISSUE TITLE
plugin-apply-destructuring changes code behavior

### ISSUE THREAD

**Issue by EvgenyOrekhov** [coderaiser/putout#50]
Before:

```js
const objectA = { property: "" };
const objectB = { property: "foo" };

const property = objectA.property || objectB.property;

console.log(property); // prints "foo" (the value from objectB)
```

After:

```js
const objectA = { property: "" };
const objectB = { property: "foo" };

const { property = objectB.property } = objectA;

console.log(property); // prints "" (the value from objectA)
```

**Comment by coderaiser** [comment]
Thank you :), fixed with f4efbd9d, landed in `@putout/plugin-apply-destructuring v5.1.1` 🎉 . 
Please re-install `putout`.

Is it works for you?

**Comment by EvgenyOrekhov** [comment]
It works, thank you!
